docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,162 @@
+* Configure single-page application routing for files outside of the *_framework* folder, which serves the default page (*index.html*) for any request that hasn't been served by a prior Static File Middleware instance.
+
+For information on ASP.NET Core app hosting and deployment, see [Host and deploy ASP.NET Core](https://docs.microsoft.com/aspnet/core/host-and-deploy).
+
+For information on deploying to Azure App Service, see the following topics:
+
+[Publish to Azure with Visual Studio](https://docs.microsoft.com/aspnet/core/tutorials/publish-to-azure-webapp-using-vs)  
+Learn how to publish an ASP.NET Core-hosted Blazor app to Azure App Service using Visual Studio.
+
+[Publish to Azure with CLI tools](https://docs.microsoft.com/aspnet/core/tutorials/publish-to-azure-webapp-using-cli)  
+Learn how to publish an ASP.NET Core app to Azure App Service using the Git command-line client.
+
+## Standalone deployment
+
+In a standalone deployment, only the Blazor client-side app is deployed to the server or hosting service. An ASP.NET Core server-side app isn't used to host the Blazor app. The Blazor app's static files are requested by the browser directly from the static file web server or service.
+
+**Rewrite URLs for correct routing**
+
+Routing requests for page components in a client-side app isn't as simple as routing requests to a server-side, hosted app. Consider a client-side app with two pages:
+
+* **_Main.cshtml_**: Loads at the root of the app and contains a link to the About page (`href="/About"`).
+* **_About.cshtml_**: About page.
+
+When the app's default document is requested using the browser's address bar (for example, `https://www.contoso.com/`):
+
+1. The browser makes a request.
+1. The default page is returned, which is usually *index.html*.
+1. *index.html* bootstraps the app.
+1. Blazor's router loads and the Razor Main page (*Main.cshtml*) is displayed.
+
+On the Main page, selecting the link to the About page loads the About page. Selecting the link to the About page works on the client because the Blazor router stops the browser from making a request on the Internet to `www.contoso.com` for `/About` and serves the About page itself. All of the requests for internal pages *within the client-side app* work the same way: Requests don't trigger browser-based requests to server-hosted resources on the Internet. The router handles the requests internally.
+
+If a request is made using the browser's address bar for `www.contoso.com/About`, the request fails. No such resource exists on the app's Internet host, so a *404 Not found* response is returned.
+
+Because browsers make requests to Internet-based hosts for client-side pages, web servers and services must rewrite all requests to the *index.html* page. When *index.html* is returned, the app's client-side router takes over and responds with the correct resource.
+
```

Comment:
Consider adding a section that discusses setting the base path here.

Reply:
Can u elaborate a little? I don't quite follow ... I'm thinking that setting the base path is host-dependent and that the *publish* folder (however it's named on the host) serves as the root of the app.